athenahealth/httpclient.go

```diff
@@ -198,6 +191,14 @@ func (h *HTTPClient) request(ctx context.Context, method, path string, body io.R
 	}
 	defer res.Body.Close()
 
+	h.logger.Info().
+		Str("method", method).
+		Str("url", reqURL).
+		Int64("bodyLength", requestBodyLength).
+		Int64("bodyContentLength", req.ContentLength).
+		Str("xRequestId", xRequestID).
+		Msg("athenahealth API request")
```

Comment:
@tcodes0 there's already a log a bit further down. If the Do() call fails, we wont see a log statement. I think we want to keep this call before the Do() but only output `		Int64("bodyLength", requestBodyLength)` in the log that occurs after Do().